client-side-js/executeControlMethod.js

```diff
@@ -42,16 +42,14 @@ async function clientSide_executeControlMethod(webElement, methodName, browserIn
                                 returnType: "none"
                             })
                         } else {
-                            // result mus be a primitive
                             if (window.wdi5.isPrimitive(result)) {
-                                // getter
                                 done({ status: 0, result: result, returnType: "result" })
                             } else if (
+                                // we have an object that is not a UI5 control
                                 typeof result === "object" &&
                                 result !== null &&
-                                // wdi5 returns a wdi5 control if the UI5 api return its control
-                                // allows method chaining
-                                !(result instanceof sap.ui.core.Control)
+                                !(result instanceof sap.ui.core.Control) &&
+                                !(result instanceof sap.ui.core.Item)
```

Comment:
Does this check also match subclasses? As sap.ui.core.ListItem extends sap.ui.core.Item. Will this also be handled correctly?